pkg/compose/compose_test.go

```diff
@@ -75,12 +79,34 @@ func TestComposeCompare(t *testing.T) {
 		composeBinsDir := t.TempDir()
 		compareDir = composeBinsDir
 		cockroachBin = filepath.Join(composeBinsDir, "cockroach")
-		err = copyBin(*flagCockroach, cockroachBin)
-		if err != nil {
+		libGeosDir = filepath.Join(composeBinsDir, "lib")
+		if err := copyBin(*flagCockroach, cockroachBin); err != nil {
 			t.Fatal(err)
 		}
-		err = copyBin(*flagCompare, filepath.Join(composeBinsDir, "compare.test"))
-		if err != nil {
+		// Copy files from flagLibGeosDir to libGeosDir.
```

Comment:
Any particular reason why we have one directory that we copy verbatim into another one (instead of just using the directory that already exists)?

Reply:
i would have expected the above comment to apply to these files:
```
// These binaries are going to be mounted as volumes when we
// start up docker-compose, but the files themselves will be
// Bazel-built symlinks. We want to copy these files to a
// different temporary location.
```

does it not?

Reply:
Oh, I see. When you're running the nightly, yes, you'll be using the Bazel cache directory with the symlinks.